app/models/user.rb

```diff
@@ -66,5 +67,10 @@ def name
     login
   end
 
-  scope :company_member, -> { where(company_member: true) }
+  scope :company_member, -> { where.not(company_member_since: nil) }
```

Comment:
yes because there will always be a time frame in which we want to get the active members. I think there is not a case for now in which we need to get only the current members. So this methods returns all users that were members at any time of history, this scope is used in ExternalPullRequestsProcessorJob, that way all prs from rs active and former members will be updated so then when a time frame is given we make sure the returned info is updated. We can change the name of the scope.